composer.json

```diff
@@ -73,5 +73,8 @@
     "require-dev": {
         "composer/composer": "1.0.*@dev",
         "phpunit/phpunit": "4.1.*"
+    },
+    "scripts": {
+        "test": "phpunit"
```

Comment:
Since PhpUnit is a dev requirement, shouldn't this point to `vendor/bin/phpunit` rather than just `phpunit`?